<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reluplexcfg</class>
 <widget class="QWidget" name="reluplexcfg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4,1">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Network File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="reluplexNetwork">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reluplexNetworkButton">
       <property name="text">
        <string>import..</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Timeout(m):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="timeoutSpin">
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Test Point:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="testpointSpin">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>3</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2,1">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Delta:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="reluplexSlider">
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="reluplexSpin">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.001000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="runWithoutSymbol">
       <property name="text">
        <string>Run Reluplex Only</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="runWithSymbol">
       <property name="text">
        <string>Run Reluplex With DeepSymbol</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
